Tests/FluentAssertions.Specs/Collections/CollectionAssertionSpecs.BeSupersetOf.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using FluentAssertions.Execution;
+using Xunit;
+using Xunit.Sdk;
+
+namespace FluentAssertions.Specs.Collections;
+
+/// <content>
+/// The BeSupersetOf specs.
+/// </content>
+public partial class CollectionAssertionSpecs
+{
+    public class BeSupersetOf
```

Comment:
We're missing test on how `BeSupersetOf` should behave when `Subject` is either `null` or empty.